src/main/java/com/google/cloud/genomics/dataflow/utils/GCSOptions.java

```diff
@@ -29,7 +29,7 @@
 import com.google.cloud.genomics.utils.GenomicsFactory;
 import com.google.common.collect.ImmutableList;
 
-import org.codehaus.jackson.annotate.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
Can more of this file go away now?  I think `Storage.Objects` is still needed but the Genomics stuff is no longer in use anywhere.